reactor-netty-core/src/test/java/reactor/netty/transport/AddressUtilsTest.java

```diff
@@ -146,6 +147,13 @@ void shouldParseAddressForIPv6WithoutBrackets() {
 		assertThat(socketAddress.getHostString()).isEqualTo("1abc:2abc:3abc:0:0:0:5abc:6abc");
 	}
 
+	@Test
+	void shouldParseAddressForIPv6WithoutBrackets_Strict() {
```

Comment:
nit: shouldn't these tests that assert an IllegalArgumentException be prefixed "shouldNotParseXxx"?

Reply:
fixed